protocol/testing/containertest/preupgrade_genesis.json

```diff
@@ -0,0 +1,2164 @@
+{
```

Comment:
Questions:
1. Could you document in this PR how this particular `.json` file was generated?
2. This file needs to for every major version right? So if we are testing `v3 -> v4` upgrade then it will need a genesis file compatible with v3. How will we manage this in the long run?

Reply:
Added better comment and PR description.

Yeah it's not ideal, but it's not the worst since we don't have to do this often. The other solution I was thinking was to check out the files at that commit, but that's also not that nice.

Reply:
For example, osmosis gets the old binary and genesis via docker hub containers that they build.